### ISSUE TITLE
Exception on 1.20.1 fabric 1.0.5+1.20

### ISSUE THREAD

**Issue by SettingDust** [sisby-folk/swingthrough#5]
```
[15:02:10] [Render thread/ERROR] (Minecraft) Unreported exception thrown!
java.lang.NoSuchMethodError: 'net.minecraft.util.shape.VoxelShape net.minecraft.block.BlockState.method_11628(net.minecraft.world.BlockView, net.minecraft.util.math.BlockPos)'
	at net.minecraft.client.render.GameRenderer.localvar$banp000$swingthrough$cacheReach(GameRenderer.java:25630) ~[client-intermediary.jar:?]
	at net.minecraft.client.render.GameRenderer.updateTargetedEntity(GameRenderer.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.tick(MinecraftClient.java:1848) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1181) ~[client-intermediary.jar:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) ~[fabric-loader-0.16.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.16.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.16.0.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
```

**Comment by sleepingdragoninn** [comment]
looks like i uploaded the wrong version there - must have been an issue with my release action.

**Comment by sleepingdragoninn** [comment]
yeah, just a bad merge caused the 1.20 version to be compiled against 1.15 - fixed in 1.0.6
